core/src/main/resources/jenkins/model/Jenkins/legend.properties

```diff
@@ -33,9 +33,9 @@ yellow=The last build was successful but unstable. \
 yellow_anime=The last build was successful but unstable. A new build is in progress.
 red=The last build failed.
 red_anime=The last build failed. A new build is in progress.
-health-81plus=Project health is over 80%. You can hover the mouse over the project’s icon for a more detailed explanation.
-health-61to80=Project health is over 60% and up to 80%. You can hover the mouse over the project’s icon for a more detailed explanation.
-health-41to60=Project health is over 40% and up to 60%. You can hover the mouse over the project’s icon for a more detailed explanation.
-health-21to40=Project health is over 20% and up to 40%. You can hover the mouse over the project’s icon for a more detailed explanation.
-health-00to20=Project health is 20% or less. You can hover the mouse over the project’s icon for a more detailed explanation.
-
+health-81plus=Project health is over 80%
+health-61to80=Project health is over 60% and up to 80%
+health-41to60=Project health is over 40% and up to 60%
+health-21to40=Project health is over 20% and up to 40%
+health-00to20=Project health is 20% or less
+iconDescription=You can hover your mouse over a project\u2019s icon for a more detailed explanation
```

Comment:
Looking at the code, there's no such thing that places a tooltip here. Comparing to 2.60.1, there's no tooltip there either. 
I'd say, more than 5 years later, we could safely remove the hover part.

Reply:
This is on the legend page this change just removes the repetition

https://ci.jenkins.io/legend
![image](https://user-images.githubusercontent.com/21194782/193588320-ba1ded15-3cbe-494d-a308-19edf22218fb.png)

Reply:
Oh I see what you mean, it doesn't do anything on this page, yeah I agree it can be removed

Reply:
This page is a legend. It explains how to use the page it is referenced on, which is list views, and it works as described there:

> <img width="615" alt="Screenshot 2022-10-04 at 09 43 37" src="https://user-images.githubusercontent.com/1831569/193762998-2d88bdcc-4347-4bba-a5e2-3dc91b91b6aa.png">

Reply:
> You can use the same argument to remove the rest of the label. There's no project here after all!

I disagree that part of the label is clear.

I don't think there's any harm in removing this, but if you think it's worth keeping the last commit could be reverted and reworded slightly to be clearer I think.